src/overlays/actors/ovl_Obj_Switch/z_obj_switch.c

```diff
@@ -5,17 +5,79 @@
  */
 
 #include "z_obj_switch.h"
+#include "objects/gameplay_dangeon_keep/gameplay_dangeon_keep.h"
 
 #define FLAGS 0x00000010
 
 #define THIS ((ObjSwitch*)thisx)
 
+#define COS_OF_5_PI_DIV_8 -0.38268343f
```

Comment:
What's everyone's thoughts on putting this in `math.h`?

Reply:
Seems weird to do for a constant specific to this actor

Reply:
If this constant appears again we can consider moving it to a math header